ckanext/switzerland/tests/test_sbb_harvester.py

```diff
@@ -109,7 +109,6 @@ def test_existing_resource_same_filename(self):
         self.assertEqual(len(dataset["resources"]), 1)
         resource = dataset["resources"][0]
 
-        self.assertEqual(resource["title"]["de"], "AAAResource")
```

Comment:
Previously, if we harvested a resource with the same filename as an existing resource, one of the fields that was copied from the old resource to the new one was its `title`. If we took over an existing resource (with a different filename) as a 'template' for the metadata, the title was not copied over, and instead the filename was used as the resource title. Obviously, in this case, we couldn't copy the title.

With this PR, we get an old resource if one is available (either with the same filename or a different one) and instead of using it as a template, we copy over some fields to the new resource. That means we construct the resource dictionary the same way whether there is an existing resource for the new filename or not. Copying over the title in only one of those cases adds complications. I also checked in the PROD website, and all resources use the filename as title anyway, so the old functionality is not being used.

In this case, I hope it's okay to just remove this test and always generate a new title based on the filename.